libraries/protocol/include/steem/protocol/smt_operations.hpp

```diff
@@ -166,6 +170,8 @@ struct smt_emissions_unit
 
 struct smt_setup_emissions_operation : public base_operation
 {
+   /// Contains Numerical Asset Identifier (NAI) of the SMT.
+   uint32_t            nai;
```

Comment:
Any cross reference to the SMT should probably be by the asset num. The NAI is a string only representation.